tests/src/whisk/core/entity/test/ViewTests.scala

```diff
@@ -229,6 +238,84 @@ class ViewTests extends FlatSpec
         getKindInPackage(pkgname2, "actions", { case (e: WhiskAction) => true case (_) => false })
         getKindInNamespaceByName(namespace2, "activations", actionName, { case (e: WhiskActivation) => (e.name == actionName) case (_) => false })
     }
+    
```

Comment:
Why not just add `getEntitiesInNamespace(namespace1)` to the previous unit test? The thought with the previous test is to check multiple views once the db is initialized. This avoids repeating tens of db operations.